packages/nocodb/src/models/Plugin.ts

```
@@ -89,36 +89,43 @@ export default class Plugin implements PluginType {
     );
 
     await NocoCache.update(`${CacheScope.PLUGIN}:${pluginId}`, updateObj);
-    await NocoCache.update(`${CacheScope.PLUGIN}:${plugin.title}`, updateObj);
 
     return this.get(pluginId);
   }
 
-  public static async isPluginActive(title: string) {
-    return !!(await this.getPluginByTitle(title))?.active;
+  public static async isPluginActive(id: string) {
+    return !!(await this.getPlugin(id))?.active;
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)